src/runtime/cli/config/configuration-fc.toml.in

```diff
@@ -32,7 +32,7 @@ valid_hypervisor_paths = @FCVALIDHYPERVISORPATHS@
 # launched in a jailed enviornment created by the jailer
 # This is disabled by default as additional setup is required
 # for this feature today.
-#jailer_path = "@FCJAILERPATH@"
+jailer_path = "@FCJAILERPATH@"
```

Comment:
This change looks good, but shouldn't we also remove the comments above the line you changed?
"""
# This is disabled by default as additional setup is required
# for this feature today.
"""
^^^^^ Those :-)